CONTRIBUTING.md

```diff
@@ -7,8 +7,12 @@ make substantive contributions, you must join the CG.
 
 Contributions to the source code repository are subject to the terms of the
 [3-Clause BSD License](./LICENSE.txt).
+**Contributions will also be exported to
+[Web Platform Tests](https://github.com/web-platform-tests/wpt)
```

Comment:
```suggestion
[web-platform-tests](https://github.com/web-platform-tests/wpt)
```

Not important, but this is how most projects members tend to refer to the project.